Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -1354,6 +1354,9 @@ void Document::set_hovered_node(Node* node)
     // https://w3c.github.io/uievents/#mouseout
     if (old_hovered_node && old_hovered_node != m_hovered_node) {
         UIEvents::MouseEventInit mouse_event_init {};
+        mouse_event_init.bubbles = true;
+        mouse_event_init.cancelable = true;
+        mouse_event_init.composed = true;
```

Comment:
I thought about adding a "create_internal()" or something, but it doesn't seem that bad to just be explicit at each callsite.